langserver/symbol.go

```diff
@@ -237,16 +238,16 @@ func (h *LangHandler) handleSymbol(ctx context.Context, conn JSONRPC2Conn, req *
 			pkgPat = h.init.RootImportPath + "/..."
 		}
 
-		var wg sync.WaitGroup
+		run := parallel.NewRun(8)
 		pkgs := buildutil.ExpandPatterns(bctx, []string{pkgPat})
 		for pkg := range pkgs {
-			wg.Add(1)
+			run.Acquire()
 			go func(pkg string) {
-				defer wg.Done()
+				defer run.Release()
 				h.collectFromPkg(bctx, fs, pkg, rootPath, &results)
 			}(pkg)
 		}
-		wg.Wait()
+		run.Wait()
```

Comment:
Intentionally ignoring the error? If so, just add a comment explaining why or use `_ = run.Wait()` to make it clear this is not an oversight.